samples/multicluster/expose-istiod-https.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istiod-gateway
+  namespace: istio-system
+spec:
+  selector:
+    istio: eastwestgateway
+  servers:
+    - port:
+        name: https-istiod
+        number: 15012
+        protocol: https
+      tls:
+        mode: SIMPLE
+        # use a valid credential here
+        credentialName: "$CREDENTIAL_NAME"
```

Comment:
if we use a varialbe here maybe we should for `ISTIOD-HOST.DOMAIN` as well. So people can do `envsubt | kubectl apply`

Reply:
I think I started with $ISTIOD-HOST but lint was complaining.  so this yaml is just a first draft, which I'll refine as the doc PR is being worked.